src/reset.css

```diff
@@ -0,0 +1,7 @@
+*,
+*:after,
+*:before {
+  margin: 0;
+	padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
use * selector only for box-sizing property. don't use it for margins and paddings

Reply:
add empty line at the end of the file